### DISCUSSION TITLE
Is it possible to output the extended config?

### DISCUSSION THREAD

**Discussion by Mitsunee** [eslint/eslint#15747] [Q&A] [answered]
I am currently working on redoing my personal configs and wanted to ask if there is a way to have eslint print out the full config with the extends field fully resolved.

**Comment by mdjermanovic** [comment, marked as answer]
You can use the [--print-config](https://eslint.org/docs/user-guide/command-line-interface#--print-config) CLI option to see the calculated config for the file passed.

**Reply by Mitsunee** [reply]
exactly what I needed, thank you 👍🏻
